### PR TITLE
Update accessibility statement

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/ETvr5FLf

### Context

The accessibility statement for the Get School Experience service needed updating to reflect the fact that we:
- had a review in 2022
- we tackled a lot of this issues back then
- we have a couple of outstanding issues left to address

### Changes proposed in this pull request

- update the statement to reflect the [approved template](https://design.education.gov.uk/accessibility/accessibility-statements/partial-statement)

### Guidance to review

- is this now reflective of the template?

### CODE REVIEW

**Review by gemmadallmandfe — COMMENTED** [review]
Looks good @MylesJarvis - have left a few minor comments

File: app/views/pages/accessibility_statement.html.erb [hunk 1]
```diff
@@ -87,65 +80,63 @@
...
         Public Sector Bodies (Websites and Mobile Applications) (No. 2)
         Accessibility Regulations 2018.
       </p>
 
       <p>
-        This service is fully compliant with the
-        <%= link_to 'Web Content Accessibility Guidelines version 2.1', 'https://www.w3.org/TR/WCAG21/' %>
-        AA standard.
+        This service is partially compliant with the
+        <%= link_to 'Web Content Accessibility Guidelines (WCAG) version 2.1', 'https://www.w3.org/TR/WCAG21/' %>
```
[the remark this answers is not in the document]
Comment (MylesJarvis) [reply app/views/pages/accessibility_statement.html.erb:105]: Apols, artefact from the last version. I thought no harm in having it, but will remove!

File: app/views/pages/accessibility_statement.html.erb [hunk 2]
```diff
@@ -87,65 +80,63 @@
...
     <section>
-      <h2 class="govuk-heading-m">Technical information about this service's accessibility</h2>
+      <h2 class="govuk-heading-m">Technical information about this website's accessibility</h2>
 
       <p>
-        DfE is committed to making its service accessible, in accordance with the
+        The Department for Education is committed to making its websites accessible, in accordance with the
         Public Sector Bodies (Websites and Mobile Applications) (No. 2)
         Accessibility Regulations 2018.
       </p>
```
[the remark this answers is not in the document]
Comment (MylesJarvis) [reply app/views/pages/accessibility_statement.html.erb:102]: Doh! Good catch, thank you

File: app/views/pages/accessibility_statement.html.erb [hunk 3]
```diff
@@ -31,54 +36,42 @@
...
-      <h2 class="govuk-heading-m">Service accessibility</h2>
-
-      <p>
-        We are not aware of any parts of our service which are not accessible.
+        <%= link_to 'AbilityNet (opens in a new tab)', 'https://mcmw.abilitynet.org.uk/', target: "_blank" %>
+        has advice on making your device easier to use if you have a disability.
       </p>
     </section>
 
     <section>
```
[the remark this answers is not in the document]
Comment (MylesJarvis) [reply app/views/pages/accessibility_statement.html.erb:44]: Thanks Gemma, my concern was the repetition like you mention. Will add this now.

File: app/views/pages/accessibility_statement.html.erb [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
-  <article class="govuk-grid-column-full">
+  <article class="govuk-grid-column-two-thirds">
     <%= page_heading do %>
-      Accessibility statement for the school experience service
+      Accessibility statement
     <% end %>
 
     <section>
+      <p>
+        This accessibility statement applies to the Get School Experience service.
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:11]:
```suggestion
        This accessibility statement applies to the Get School Experience website.
```

File: app/views/pages/accessibility_statement.html.erb [hunk 5]
```diff
@@ -1,12 +1,16 @@
...
+      </p>
+
       <p>
         This service is run by the Department for Education (DfE).
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:15]:
```suggestion
        This website is run by the Department for Education (DfE).
```

File: app/views/pages/accessibility_statement.html.erb [hunk 6]
```diff
@@ -1,12 +1,16 @@
...
       </p>
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:16]: Also line 15/19 - replace `service` with `website`

File: app/views/pages/accessibility_statement.html.erb [hunk 7]
```diff
@@ -19,66 +23,59 @@
       <ul class="govuk-list govuk-list--bullet">
         <li>change colours, contrast levels and fonts</li>
         <li>zoom in up to 300% without the text spilling off the screen</li>
-        <li>navigate most of the service using just a keyboard and speech recognition software</li>
+        <li>navigate most of the service using just a keyboard</li>
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:26]:
```suggestion
        <li>navigate most of the website using just a keyboard</li>
```

File: app/views/pages/accessibility_statement.html.erb [hunk 8]
```diff
@@ -19,66 +23,59 @@
...
+        <li>navigate most of the website using speech recognition software</li>
         <li>
           listen to most of the service using a screen reader (including the most
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:29]:
```suggestion
          listen to most of the website using a screen reader (including the most
```

File: app/views/pages/accessibility_statement.html.erb [hunk 9]
```diff
@@ -19,66 +23,59 @@
...
+        We'll consider your request and get back to you in 5 working days.
       </p>
     </section>
 
     <section>
-      <h2 class="govuk-heading-m">Reporting accessibility problems with this service</h2>
+      <h2 class="govuk-heading-m">Reporting accessibility problems with this website</h2>
 
       <p>
         We're always looking to improve the accessibility of this service.
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:73]:
```suggestion
        We're always looking to improve the accessibility of this website.
```

File: app/views/pages/accessibility_statement.html.erb [hunk 10]
```diff
@@ -87,65 +84,64 @@
...
-          'http://www.legislation.gov.uk/uksi/2018/852/contents/made' %>
+        The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
+        Public Sector Bodies (Websites and Mobile Applications)
+        (No. 2) Accessibility Regulations 2018 (the 'accessibility regulations').
       </p>
 
       <p>
         If you're not happy with how we respond to your complaint, contact the
-        <%= link_to 'Equality Advisory and Support Service (EASS)', 'https://www.equalityadvisoryservice.com/' %>.
+        <%= link_to 'Equality Advisory and Support Service (opens in a new tab)', 'https://www.equalityadvisoryservice.com/', target: "_blank" %>.
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:94]:
```suggestion
        <%= link_to 'contact the Equality Advisory and Support Service (opens in a new tab)', 'https://www.equalityadvisoryservice.com/', target: "_blank" %>.
```

Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:94]:
```suggestion
        <%= link_to 'contact the Equality Advisory and Support Service (opens in a new tab)', 'https://www.equalityadvisoryservice.com/', target: "_blank" %>.
```

File: app/views/pages/accessibility_statement.html.erb [hunk 11]
```diff
@@ -87,65 +84,64 @@
...
-        enforcing the 'accessibility regulations' - officially known as the
-        <%= link_to 'Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.',
-          'http://www.legislation.gov.uk/uksi/2018/852/contents/made' %>
+        The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
+        Public Sector Bodies (Websites and Mobile Applications)
+        (No. 2) Accessibility Regulations 2018 (the 'accessibility regulations').
       </p>
 
       <p>
         If you're not happy with how we respond to your complaint, contact the
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:93]:
```suggestion
        If you're not happy with how we respond to your complaint, 
```

File: app/views/pages/accessibility_statement.html.erb [hunk 12]
```diff
@@ -87,65 +84,64 @@
...
       </p>
+    </section>
 
+    <section>
+      <h2 class="govuk-heading-m">Compliance status</h2>
       <p>
-        This service is fully compliant with the
-        <%= link_to 'Web Content Accessibility Guidelines version 2.1', 'https://www.w3.org/TR/WCAG21/' %>
-        AA standard.
+        This service is partially compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard, due to the following non-compliances:
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:111]:
```suggestion
        This website is partially compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard, due to the following non-compliances:
```

File: app/views/pages/accessibility_statement.html.erb [hunk 13]
```diff
@@ -87,65 +84,64 @@
...
-      </h3>
-
       <p>
-        We have not identified any problems relating to interactive tools and
-        transactions.
+         <ul class="govuk-list govuk-list--bullet">
+        <li>if a candidate tries to search for a school experience placement without entering a postcode,
+          we do not surface an error message.
+          This does not meet <%= link_to 'WCAG version 2.1 A – 3.3.1 Error Identification (opens in a new tab)', 'https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html' %>
+          and <%= link_to 'AA – 3.3.3 Error Suggestion', 'https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html', target: "_blank" %>.</li>
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:118]:
```suggestion
          and <%= link_to 'WCAG version 2.1 AA – 3.3.3 Error Suggestion', 'https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html', target: "_blank" %>.</li>
```
I would add this in again to be really clear - especially given that WCAG 2.2 is around the corner

File: app/views/pages/accessibility_statement.html.erb [hunk 14]
```diff
@@ -87,65 +84,64 @@
...
+      <h2 class="govuk-heading-m">What we're doing to improve accessibility</h2>
       <p>
-        This service was last tested on 3 July 2019. The test was carried out by
-        the <%= link_to 'Digital Accessibility Centre (DAC)', 'https://digitalaccessibilitycentre.org/' %>.
+        We are currently working to address the known accessibility issues.
       </p>
-
       <p>
-        We tested our whole service platform.
+        We’ll continue to test the accessibility of this service as it develops.
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility_statement.html.erb:132]:
```suggestion
        We’ll continue to test the accessibility of this website as it develops.
```

**Review by gemmadallmandfe — COMMENTED** [review]
@MylesJarvis Realised that we talk about service rather than website in loads of other places - sorry, missed this before!

Let's get this out on Thursday

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**